### PR TITLE
gh-release: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

## Tasks

- quote `tag` placeholders
- use range for ellipsis (experimental, #9742)

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages/common/gh-release.md [hunk 1]
```diff
@@ -9,20 +9,20 @@
...
 
 - Download assets from a specific release:
 
-`gh release download {{tag}}`
+`gh release download "{{tag}}"`
 
 - Upload assets to a specific release:
 
-`gh release upload {{tag}} {{path/to/file1}} {{path/to/file2}}`
+`gh release upload "{{tag}}" {{path/to/file1 path/to/file2 ...(1..)}}`
```
Comment (navarroaxel) [inline pages/common/gh-release.md:28]: What means `...(1..)`?
```suggestion
`gh release upload "{{tag}}" {{path/to/file1 path/to/file2 ...}}`
```
Why tldr is adding Calculus notation? :laughing: these pages should be easy to read for non-college people. We already have a lot of formal language used to explain examples.

Comment (kbdharun) [reply pages/common/gh-release.md:28]: > What means `...(1..)`?
> ```suggestion
> `gh release upload "{{tag}}" {{path/to/file1 path/to/file2 ...}}`
> ```
> Why tldr is adding Calculus notation? :laughing: these pages should be easy to read for non-college people. We already have a lot of formal language used to explain examples.

Exactly, I was confused too.

Comment (EmilySeville7cfg) [reply pages/common/gh-release.md:28]: > Why tldr is adding Calculus notation? laughing these pages should be easy to read for non-college people. We already have a lot of formal language used to explain examples.

It's a formalized syntax for ellipsis that supports ranges. In this case it means to expect 1 or more files here. As now there is no standardized way to present how much arguments are expected to ellipsis.

Comment (navarroaxel) [reply pages/common/gh-release.md:28]: The user should be able to read pages without learn a syntax guide. I'm not able to understand `...(n...)` without read the guide.

Comment (EmilySeville7cfg) [reply pages/common/gh-release.md:28]: I think I'll change this syntax to another one as it causes confusion as I got.

Comment (EmilySeville7cfg) [reply pages/common/gh-release.md]: Please rereview this line again. :)

File: pages/common/gh-release.md [hunk 2]
```diff
@@ -9,20 +9,20 @@
 
 - Display information about a specific release:
 
-`gh release view {{tag}}`
+`gh release view "{{tag}}"`
```
Comment (sbrl) [inline pages/common/gh-release.md:12]: This syntax suggests that double quotes around the tag name are required, which doesn't appear to be the case?

File: pages/common/gh-release.md [hunk 3]
```diff
@@ -9,20 +9,20 @@
...
 
 - Create a new release:
 
-`gh release create {{tag}}`
+`gh release create "{{tag}}"`
```
Comment (sbrl) [inline pages/common/gh-release.md:16]: Same here.

File: pages/common/gh-release.md [hunk 4]
```diff
@@ -9,20 +9,20 @@
...
 
 - Delete a specific release:
 
-`gh release delete {{tag}}`
+`gh release delete "{{tag}}"`
 
 - Download assets from a specific release:
 
-`gh release download {{tag}}`
+`gh release download "{{tag}}"`
```
Comment (sbrl) [inline pages/common/gh-release.md:24]: Same here.

File: pages/common/gh-release.md [hunk 5]
```diff
@@ -9,20 +9,20 @@
...
 
 - Create a new release:
 
-`gh release create {{tag}}`
+`gh release create "{{tag}}"`
 
 - Delete a specific release:
 
-`gh release delete {{tag}}`
+`gh release delete "{{tag}}"`
```
Comment (sbrl) [inline pages/common/gh-release.md:20]: Same here.

File: pages/common/gh-release.md [hunk 6]
```diff
@@ -9,20 +9,20 @@
...
 
 - Download assets from a specific release:
 
-`gh release download {{tag}}`
+`gh release download "{{tag}}"`
 
 - Upload assets to a specific release:
 
-`gh release upload {{tag}} {{path/to/file1}} {{path/to/file2}}`
+`gh release upload "{{tag}}" {{1 or more: path/to/file1 path/to/file2 ...}}`
```
Comment (sbrl) [inline pages/common/gh-release.md:28]: Same here.

Also, the `1 or more: ` would be confusing for beginners, since they would just type out those literal words and wonder why it didn't work. An alternative:
```suggestion
`gh release upload {{tag}} {{path/to/file1}} {{path/to/file2}} {{...}}`
```

tldr pages should be intuitive to read, not prescripted to a single style with technical syntax. The point of tldr-pages is to be easier to read and more intuitive than regular man pages. To this end, it is inevitable that tldr pages will all look  little different to one another. While fixing a few errors is good, ensuring all pages are rigorously adhering to a single style is not good, since each page has it's own circumstances which demand it be different to the rest..... even within the subcommands of a single command.

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page! I've left some comments below for you to review.

File: pages/common/gh-release.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/common/gh-release.md:12]: I don't know whether tags can contain spaces that's why I put quotes here.

File: pages/common/gh-release.md (hunk 6 above)
Comment (EmilySeville7cfg) [reply pages/common/gh-release.md]: `{{...}}` is incorrect syntax for ellipsis, the correct one is always just this: `{{placeholder1 placeholder2 ...}}`.

Comment (EmilySeville7cfg) [reply pages/common/gh-release.md]: I've removed `1 or more`.
